src/ui/templates/partials/phase-banner.njk

```diff
@@ -1,5 +1,9 @@
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
 
+{% set bannerHtml %}
+  <span data-cy="phase-banner-text">This is a new service - your <a class="govuk-link" href="{{ FEEDBACK_ROUTE }}" data-cy="phase-banner-feedback-link">feedback</a> will help us to improve it.</span>
```

Comment:
Whilst we're in here could we have this text coming from `CONTENT_STRINGS`? I think it's the only thing that doesn't come from content strings 😢